private/buf/cmd/buf/command/beta/registry/plugin/pluginpush/pluginpush.go

```diff
@@ -421,6 +421,9 @@ func findExistingDigestForImageID(
 	if err != nil {
 		structuredErr := new(transport.Error)
 		if errors.As(err, &structuredErr) {
+			if structuredErr.StatusCode == http.StatusUnauthorized {
+				return "", errors.New("you are not authenticated. For details, visit https://buf.build/docs/bsr/authentication")
```

Comment:
We'll probably want to update all the https://docs.buf.build references in the future, @rakuista is this on your radar. Do we care?